forLoopTimeout/forLoopTimeout.js

```diff
@@ -5,9 +5,16 @@
 // I've been asked this three times in separate interviews.
 
 for (var i = 1; i <= 10; i++) {
-	setTimeout(function() {
+	(function(i) {
```

Comment:
It seems like you have some inconsistency with your code indentation. In previous files I've seen you go with 1/2 the distance on the spacing that you have on this file.